File: MultiplicationTable/MultiplicationTable.js
const prompt = require("prompt-sync")(); // npm i prompt-sync

multiplicationTable();
function multiplicationTable() {
	console.log("-- Multiplication Table --");

	const times = 10;
	const num = Number(prompt("Number: "));

	for (let i = 0; i < times + 1; i++) {
		console.log(num + " * " + i + " = " + (i * num));
	}
}

File: MultiplicationTable/MultiplicationTable.py
print("-- Multiplication Table --")

newNumber = "Y"

while newNumber == "Y":
    print("-"*20)
    num = int(input("Number ➜ "))
    for x in range(11):
        print(f"{x} * {num} = {x * num}")

    newNumber = input("New number [Y/N]: ")
